org.eclipse.wb.core.java/src/org/eclipse/wb/internal/core/nls/bundle/AbstractBundleSource.java

```diff
@@ -393,7 +404,8 @@ public final void apply_externalizeProperty(GenericProperty property, String key
    * Replace given expression with externalized code.
    */
   protected abstract BasicExpressionInfo apply_externalize_replaceExpression(
-      GenericProperty property, String key) throws Exception;
+      GenericProperty property,

```

Comment:
This change does not seem neccesary

Reply:
This was my Eclipse auto-correct, not intentional